config/environments/_shared_.ts

```diff
@@ -346,6 +346,19 @@ export const EternumGlobalConfig: Config = {
   realmUpgradeCosts: REALM_UPGRADE_COSTS,
   realmMaxLevel: REALM_MAX_LEVEL,
   villageMaxLevel: VILLAGE_MAX_LEVEL,
+  questGames: [
+    {
+      address: "0x01e1c477f2ef896fd638b50caa31e3aa8f504d5c6cb3c09c99cd0b72523f07f7",
```

Comment:
use a const, what is this address supposed to represent ?

Reply:
this is the darkshuffle game address, have given the appropriate comment and moved into consts